test/e2e/petset.go

```diff
@@ -53,8 +55,33 @@ const (
 	// Should the test restart petset clusters?
 	// TODO: enable when we've productionzed bringup of pets in this e2e.
 	restartCluster = false
+	add            = "ADD"
```

Comment:
i wasn't able to find relevant events declared in api, do you have something specific in mind?